ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__ota-multi-stable-4.20-upgrade-from-stable-4.20.yaml

```diff
@@ -0,0 +1,103 @@
+base_images:
+  ansible:
+    name: "4.20"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.20"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.20"
+    namespace: ocp
+    tag: openstack-installer
+  tools:
+    name: "4.20"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.20"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+releases:
+  intermediate1:
+    prerelease:
+      architecture: multi
+      product: ocp
+      relative: 1
+      version_bounds:
+        lower: 4.20.0-0
+        stream: 4-stable
+        upper: 4.21.0-0
+  latest:
+    release:
+      architecture: multi
+      channel: candidate
+      version: 4.20.0-ec.0
+  target:
+    release:
+      architecture: multi
+      channel: candidate
+      version: "4.20"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: baremetal-ipi-retarget-f28
+  capabilities:
+  - intranet
+  cron: 53 15 4 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
+      RESERVE_BOOTSTRAP: "false"
+      architecture: amd64
+      masters: "3"
+      workers: "2"
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${INTERMEDIATE_RETARGET}" > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: INTERMEDIATE_RETARGET
+        name: release:intermediate1
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - ref: cucushift-upgrade-retarget
+    - ref: cucushift-upgrade-healthcheck
+    workflow: baremetal-lab-ipi
+- as: gcp-ipi-retarget-f28
+  cron: 20 20 22 * *
+  steps:
+    cluster_profile: gcp-qe
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${INTERMEDIATE_RETARGET}" > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: INTERMEDIATE_RETARGET
+        name: release:intermediate1
```

Comment:
If you use intermediate1 in `gcp-ipi-retarget-f28`, then the tests is duplicate with `baremetal-ipi-retarget-f28`.
But anyway, you can temporarily using intermediate1, but don't forget update it to intermediate2 when there are OCP 4.21 payloads.

intermediate1 meant to test: 4.20.0 -> 4.20.n -> 4.20.(n+1)
but intermediate2 meant to test: 4.20.0 -> 4.20.latest -> 4.21